#!/usr/bin/python3

def raise_exception_msg(message=""):
    """
    Print raise_name with message
    """

    raise NameError(message)
